app/serializers/rent_index_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class RentIndexSerializer < ActiveModel::Serializer
+  attributes :id, :from, :to, :book, :user
+
+  def user
+    UserSerializer.new(User.find(object.user_id))
```

Comment:
If we `belongs_to :user` in rent model, we can add also here a `belongs_to` and the Serializer will do the magic of serializing for you.